### PR TITLE
Make dryrun the default

### LINKED ISSUE #1333: Make dryrun the default
So we don't risk making mistakes while testing a new tool (both us and, especially, new contributors).

### PR DESCRIPTION
**Pull request by marco-c**
And add a "--no-dryrun" option to actually make changes

Fixes #1333

### CODE REVIEW

File: auto_nag/multinaggers.py [hunk 1]
```diff
@@ -33,9 +33,9 @@ def get_args_parser(self):
         parser = argparse.ArgumentParser(description=self.description())
         parser.add_argument(
             "-d",
```
Comment (suhaibmujahid) [inline auto_nag/multinaggers.py:35]: Here we keep the short flag to `-d` (not consistent). I feel a better flag name could be `--production` with no short flag version (we don't want to make it easier to do a mistake).

**Review by suhaibmujahid — APPROVED** [review]
It is an excellent idea to set the dray run as the default mode.

File: auto_nag/multinaggers.py (hunk 1 above)
Comment (marco-c) [reply auto_nag/multinaggers.py:35]: Agreed, I'll change "--no-dryrun" to "production" (I'll keep the "dryrun" name for the variable just so we don't have to modify everything everywhere).

File: auto_nag/bzcleaner.py [hunk 2]
```diff
@@ -623,11 +623,11 @@ def get_args_parser(self):
         """Get the argumends from the command line"""
         parser = argparse.ArgumentParser(description=self.description())
         parser.add_argument(
-            "-d",
```
Comment (suhaibmujahid) [inline auto_nag/bzcleaner.py:626]: Do we need a short flag version (`-p`)? I think having the full version is enough. We do not want to make it handy.

**Review by suhaibmujahid — APPROVED** [review]
LGTM.

File: auto_nag/bzcleaner.py [hunk 3]
```diff
@@ -623,11 +623,11 @@ def get_args_parser(self):
...
-            "--dryrun",
+            "-p",
```
Comment (suhaibmujahid) [inline auto_nag/bzcleaner.py:626]: Do we need a short flag version (-p)? I think having the full version is enough. We do not want to make it handy.

**Pull request merged**